### PR TITLE
feat: Finish InlineForm style changes (ACQ-2377)

### PR DESCRIPTION
**Pull request by alejandraarri**

### CODE REVIEW

**Review by 0ubbe — APPROVED** [review]
Nice, much cleaner 🙌🏽

File: packages/pci-sdk-iframe/src/pages/app/components/InlineForm/InlineForm.tsx [hunk 1]
```diff
@@ -17,13 +17,13 @@ export default function InlineForm(props: IInlineFormProps) {
 	return (
 		<form
 			className="InlineForm"
-			style={props.theme.inlineForm}
+			style={props.theme.inlineForm?.container}
```
Comment (IagoLast) [inline packages/pci-sdk-iframe/src/pages/app/components/InlineForm/InlineForm.tsx:20]: what happens if we don't pass an inlineForm in the theme. Does this throws? We had a similar issue in a previous version 🤔

**Review by IagoLast — COMMENTED** [review]
- We need to force a theme without certain fields to ensure this wont crash.
- Does the extend theme service works ok with the new changes?

Otherwise LGTM  🚀

File: packages/pci-sdk-iframe/src/pages/app/components/InlineForm/InlineForm.tsx (hunk 1 above)
Comment (alejandraarri) [reply packages/pci-sdk-iframe/src/pages/app/components/InlineForm/InlineForm.tsx:20]: This just doesn’t add the class to the form. In that case, if you don't extend any of our themes the border is going to be the default border of the button.

**Comment by alejandraarri** [comment]
> * We need to force a theme without certain fields to ensure this wont crash.

Why?? InlineForm can be undefined, right? In that case, the classname is not passed to the HTML element.

> * Does the extend theme service works ok with the new changes?

Yes, it does. I tested this, extending both themes and without extending any theme.

**Pull request merged**